### ISSUE TITLE
lqt_seek_video not being used

### ISSUE THREAD

**Issue by agraef** [umlaeute/Gem#238]
Maybe this is just old cruft not being utilized any more (then it's probably better to remove it), or maybe it's intentional. But right now `lqt_seek_video` isn't being used at all. Looking at: https://github.com/umlaeute/Gem/blob/1ab32e47ec2362fc6257c574abb31c539d5204ac/plugins/filmQT4L/filmQT4L.cpp#L191

Here's the code:

~~~c++
#ifdef HAVE_QUICKTIME_SEEK_VIDEO
  lqt_seek_video        (m_quickfile, m_curTrack, m_curFrame, m_curTrack);
#else
  if ((quicktime_set_video_position(m_quickfile, m_curFrame,
                                      m_curTrack))) {  }
#endif
  return film::SUCCESS;
~~~

There are a number of issues here:

1. The config.h symbol to test for is `HAVE_LQT_SEEK_VIDEO`, not `HAVE_QUICKTIME_SEEK_VIDEO`.

2. The lqt.h header needs to be included in the filmQT4L.h file to get the declaration of this function.

3. `lqt_seek_video` takes three parameters, not four.

4. If you're using that function, maybe you should be using `lqt_video_duration` in lieu of `quicktime_video_length` as well (at least that's what the libquicktime doxygen docs indicate).

I'm not saying that this *should* be enabled, probably it's better to keep it the way it is, in order to not break this backend. It's just something I came across while studying the QT4L source code, and left me scratching my head wondering whether this was supposed to be working.

**Comment by umlaeute** [comment]
:-) good catch. it's been lingering for 9 years :-)
PR for fixing the issue is welcome

**Comment by umlaeute** [comment]
Closed with 1ab32e47...6a8ae7dd
